### PR TITLE
[CI:DOCS] Clarify the need for qemu-user-static package

### PR DESCRIPTION
**Pull request by rhatdan**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@nalind @TomSweeneyRedHat @vrothberg @giuseppe PTAL

File: docs/buildah-build.1.md [hunk 1]
```diff
@@ -621,9 +621,7 @@ the `arch` command.  Valid OS and architecture name combinations are listed as
 values for $GOOS and $GOARCH at https://golang.org/doc/install/source#environment,
 and can also be found by running `go tool dist list`.
 
-While `buildah bud` is happy to use base images and build images for any
-platform that exists, `RUN` instructions will not be able to succeed without
-the help of emulation provided by packages like `qemu-user-static`.
+The `buildah build` command allows building images for all Linux architectures, even non nativ architectures. When building images for a different architecture then the local architecture the `RUN` instructions require emulation software installed on the host provided by packages like `qemu-user-static`. Note: it is always preferred to build images on the native architecture if possible.
```
Comment (vrothberg) [inline docs/buildah-build.1.md:624]:
```suggestion
The `buildah build` command allows building images for all Linux architectures, even non-native architectures. When building images for a different architecture,  the `RUN` instructions require emulation software installed on the host provided by packages like `qemu-user-static`. Note: it is always preferred to build images on the native architecture if possible.
```

**Review by vrothberg — APPROVED** [review]

**Review by vrothberg — APPROVED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**